src/Job/components/ContentBox/ContentBox.ts

```diff
@@ -0,0 +1,33 @@
+import { Box } from '@yousty/ui-kit';
+import styled, { css } from 'styled-components';
+
+export type BoxSize = 'small' | 'large';
+
+type BoxDimensions = {
+  [key in BoxSize]: {
+    x: number;
+    y: number;
+  };
+};
+
+const resultBoxDimensions: BoxDimensions = {
+  small: {
+    x: 4,
+    y: 4,
+  },
+  large: {
+    x: 14,
+    y: 7,
+  },
+};
+
+export const getBoxSize = (width: number): BoxSize => (width <= 768 ? 'small' : 'large');
+
+export default styled(Box)<{ size: BoxSize }>`
```

Comment:
To save a bit on the component, I suggest moving to `div` instead of `Box`
```suggestion
export default styled.div<{ size: BoxSize }>`
```